src/main/java/com/minecolonies/network/messages/AssignFieldMessage.java

```diff
@@ -69,6 +70,12 @@ public IMessage onMessage(AssignFieldMessage message, MessageContext ctx)
         final Colony colony = ColonyManager.getColony(message.colonyId);
         if (colony != null)
         {
+            //Verify player has permission to do edit permissions
+            if(!colony.getPermissions().hasPermission(ctx.getServerHandler().playerEntity, Permissions.Action.SEND_MESSAGES))
```

Comment:
That might be has permission to get citizens conversations. the OPEN_HUTS permission is usually a pretty good one.